feature/links/application/service/linkActionService.ts

```
@@ -95,6 +97,32 @@
+  ) {
+    // useTopViewLinksのキャッシュをクリア
+    mutate(["today-links", userId]);
+
+    // その他の関連するキャッシュもクリア
+    mutate(["swipeable-links", userId]);
+    mutate([`user-links-${userId}`, 10]); // デフォルトのlimit値を使用
+
+    // 汎用的なキャッシュもクリア
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    mutate(
+      (key: any) =>
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)